### DISCUSSION TITLE
v4.6.0 (Beta)

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#6715] [Releases]
## Highlights
* **QEMU v9.1.0** The backend has been updated to the latest upstream QEMU which brings with it a variety of bug fixes and performance improvements.
* (macOS 15) **Nested virtualization for Linux** Linux VMs using Apple Virtualization backend on macOS 15 and M3 or newer will now have nested virtualization enabled by default.
* (macOS 15) **Total Store Ordering for QEMU** VMs using QEMU backend on macOS 15 can now enable TSO (in QEMU settings). When TSO is enabled on a guest operating system that is aware of the system register (i.e. Rosetta for Linux), performance of Intel emulation within the guest can be greatly improved.

## Notes
* There is a known issue amongst some users with UTM running on macOS 15 where accesses to files will fail due to permission error. We are still investigating this (#6639) and related issues. In the meantime, we do not recommend upgrading to macOS 15.
* The update to QEMU v9.1.0 will likely introduce new issues and regressions. If you experience any problem (crashes, performance, errors) that was not evident in v4.5.x, please [open an issue](https://github.com/utmapp/UTM/issues) and provide all the requested information. It would also be helpful if you can provide a reproducible example and note the last version of UTM where it worked properly.

## Changes (v4.6.0)
* Updated to QEMU v9.1.0
* Fixed "RPC timeout" when stopping a VM
* Fixed a race condition when cleaning up SPICE objects
* Localization: Updated Chinese (Hong Kong, Simplified) (thanks @changanmoon)
* (iOS 16+) Home: Always show the sidebar
* (macOS 15) Enable nested virtualization when available

## Issues
Please check the full list on [Github](https://github.com/utmapp/UTM/issues) and help report any bug you find that is not listed.

## Installation

* [iOS](https://docs.getutm.app/installation/ios/)
* [macOS](https://docs.getutm.app/installation/macos/)

| File | Description | Installation | JIT | Hypervisor | USB |
|------|------------|--------------|-----|-----------|-----|
| UTM.deb | Jailbroken iOS version | Open in Cydia, dpkg, or Sileo | Yes | Yes(1) | Yes |
| UTM.dmg | macOS version | Mounting and copying UTM.app to /Applications | Yes | Yes | Yes |
| UTM.ipa | Non-jailbroken iOS version (sideloading) | AltStore, etc (see guide) | Yes(2) | No | No |
| UTM-HV.ipa | Non-jailbroken iOS version (TrollStore) | TrollStore | Yes | Yes(1) | Yes |
| UTM-SE.ipa | Non-jailbroken iOS version (sideloading) | AltStore, enterprise signing, etc | No | No | No |
| UTM-Remote.ipa | Remote client | Any | No | No | No |

1. Hypervisor on iOS requires an M1 iPad or newer.
2. Enabling JIT may require a separate JIT enabler such as [Jitterbug][2] or Jitstreamer.

[1]: https://getutm.app/install/
[2]: https://github.com/osy/Jitterbug

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v4.6.0'>v4.6.0 (Beta)</a>.</em>

**Comment by ylluminate** [comment]
This is great to see! Thank you!

**Reply by trukhinyuri** [reply]
Wait nested virtualization for windows for WSL2!

**Comment by ryne007** [comment]
![image](https://github.com/user-attachments/assets/4a21ed64-9c5b-4850-ae79-34377d258e98)

egrep -c '(vmx|svm)' /proc/cpuinfo returns 0

it shows nested virtualization not supported on the lastest UTM beta..
I have tried this on M4 pro using ubuntu 24.04 iso
